daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/NumericModule.scala

```diff
@@ -35,6 +35,15 @@ abstract class NumericModule {
     */
   val maxPrecision = 38
```

Comment:
As discussed, max scale should be 37

```suggestion
   val maxPrecision = 37
```

Reply:
I will do it in a subsequent PR.

Reply:
^ ah, I misunderstood. Yes, that's right.